common/src/main/java/com/wynntils/wynn/utils/WynnItemUtils.java

```diff
@@ -91,8 +91,9 @@ public static ItemIdentificationContainer identificationFromLore(Component lore,
      */
     public static ItemIdentificationContainer identificationFromValue(
             Component lore, ItemProfile item, String idName, String shortIdName, int value, int starCount) {
-        boolean isInverted = IdentificationOrderer.INSTANCE.isInverted(shortIdName);
         IdentificationProfile idProfile = item.getStatuses().get(shortIdName);
+        boolean isInverted =
+                idProfile != null ? idProfile.isInverted() : IdentificationOrderer.INSTANCE.isInverted(shortIdName);
```

Comment:
What is the difference between idProfile and the other?

Reply:
the second case is intended to deal with items that aren't in the API for whatever reason and thus have no profile attached to them. it's not really necessary to do it like this, but it mirrors the check for the ID type in the adjacent line